api/routes/api_routes.py

```diff
@@ -61,42 +57,39 @@ def get_pmi(id):
 @api.route('/api/v1/pmi', methods=['GET'])
 def get_all_pmi():
     return pmicontroller.index()
-
-@api.route('/api/v1/pmi', methods=['POST'])
-def add_pmi():
-    return pmicontroller.add_pmi()
-#
-@api.route('/api/v1/pmi/<id>', methods=['PUT'])
-def update_pmi(id):
-    return pmicontroller.update_pmi(id)
 #
-@api.route('/api/v1/pmi/<id>', methods=['DELETE'])
-def destroy_pmi(id):
-    return pmicontroller.destroy_pmi(id)
+# @api.route('/api/v1/pmi', methods=['POST'])
+# def add_pmi():
+#     return pmicontroller.add_pmi()
+# #
+# @api.route('/api/v1/pmi/<id>', methods=['PUT'])
+# def update_pmi(id):
+#     return pmicontroller.update_pmi(id)
+# #
+# @api.route('/api/v1/pmi/<id>', methods=['DELETE'])
+# def destroy_pmi(id):
+#     return pmicontroller.destroy_pmi(id)
 
 @api.route('/api/v1/property-tax/<id>', methods=['GET'])
 def get_property_tax(id):
     return propertytaxcontroller.get_property_tax(id)
 
-@api.route('/api/v1/property-tax', methods=['POST'])
-def add_property_tax():
-    return propertytaxcontroller.add_property_tax()
-
 @api.route('/api/v1/property-tax', methods=['GET'])
 def all_property_tax():
     return propertytaxcontroller.index()
 
-@api.route('/api/v1/property-tax/<id>', methods=['PUT'])
-def update_property_tax(id):
-    return propertytaxcontroller.update_property_tax(id)
+# @api.route('/api/v1/property-tax', methods=['POST'])
```

Comment:
I see we are commenting these routes until Iteration2 - where we have admin authorization on these routes